### PR TITLE
Add MercadoLibre provider

### PR DESCRIPTION
**Pull request by MarianoMoreyra**
This PR adds support for [MercadoLibre](http://mercadolibre.com/), an eCommerce platform from Argentina that has presence in 18 Latin American countries.

Hope the platform (and this PR) meets the requirements!

### CODE REVIEW

**Comment by MarianoMoreyra** [comment]
Forgot to run php-cs-fix! Sorry about that!

File: src/MercadoLibre/Provider.php [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    const IDENTIFIER = 'MERCADOLIBRE';
+    const DOMAIN = [
```
Comment (atymic) [inline src/MercadoLibre/Provider.php:14]: space between these lines please

File: src/MercadoLibre/Provider.php [hunk 2]
```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace SocialiteProviders\MercadoLibre;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
```
Comment (atymic) [inline src/MercadoLibre/Provider.php:8]: Could you add an `@see` docblock with a link to the provider's docs here please?

**Review by atymic — APPROVED** [review]

**Comment by atymic** [comment]
Looks good thanks, just a couple of super minor things.

**Comment by MarianoMoreyra** [comment]
Thanks to you! Added your suggestions with my last commit.

**Comment by atymic** [comment]
https://packagist.org/packages/socialiteproviders/mercadolibre

**Pull request merged**